lib/api/2.0/notification.js

```diff
@@ -30,7 +32,45 @@ var notificationProgressPost = controller(function(req, res){
     });
 });
 
+/**
+ * @api {post} /api/2.0/notification/alerts
+ * @apiDescription Receives Redfish alerts, refactors the message and publishes it into amqp
+ * @apiName notification post
+ * @apiGroup notification
+ */
+var twistedParser = parser.json({ type: 'application/x-www-form-urlencoded' });
+var jsonParser = parser.json();
+var notificationAlertsPost = controller({success: 201}, function(req,res) {
+    var format= req.headers["content-type"];
+    return Promise.try(function(){
+        return new Promise(function(resolve, reject) {
+            if (format === 'application/x-www-form-urlencoded'){
+                //This is a work arround where some vendors are sending json data
+                // with http header content-type of “x-www-form-urlencoded”
+                twistedParser(req, res, function (err) {
+                    if (err) {
+                        return reject(err);
+                    }
+                    resolve();
+                });
+            }
+            else{
```

Comment:
@tannoa2 I don't think this else case is needed for `application/json`.  The swagger params parser should handle this case.